### DISCUSSION TITLE
No apk under releases since v4.4.1

### DISCUSSION THREAD

**Discussion by Andrewskiz** [Emurgo/yoroi-mobile#2334] [General]
Hey, it would be great if you could provide an apk for current releases of Yoroi. I am still on version 4.4.1 since that is the last apk  available.

**Comment by Nebyt** [comment]
Hey @Andrewskiz. Thank you fro your report.
I have checked the Google Play store and I see the latest version of app is 4.9.2 as it should be.
https://play.google.com/store/apps/details?id=com.emurgo&hl=en
![image](https://user-images.githubusercontent.com/22872852/218400313-b30a2179-d21f-4aea-b35f-6b35c665181a.png)

**Reply by Andrewskiz** [reply]
Unfortunately, I can't use the play store because I don't have the Google Play services on my phone or a Google account. I use the F-Droid store for most of my apps, and Obtanium for the rest that are on Github. Providing an apk would be a great help for those of us who can't / won't use Google services for privacy and security reasons.

**Comment by scottbetza** [comment]
Agree with @Andrewskiz. If you're not going to use F-Droid, post the APKs to releases.
